cypari2/closure.pyx

```diff
@@ -1,3 +1,5 @@
+# distutils: include_dirs = INCLUDE_DIRS
+# distutils: library_dirs = LIBRARY_DIRS
```

Comment:
What problem is this trying to solve?

Reply:
Is the information in setup files enough for Cython?

Reply:
Yes, setuptools already passes these directories to the C compiler/linker

Reply:
good. should be fixed by 2ef1a25